### PR TITLE
feat: import scene from scene feature

### PR DESCRIPTION
**Pull request by theophile-jr**
Import scene from scene "Import a scene in a scene (2)" on project's pdf.

`imports : {
    scenes = (
        {path = "tests/subject3.cfg"}, #creating a loop, handled by the parser.
    );
};
`

It handles loop (import subject3.cfg from subject3.cfg) or multiple import (import multiple times subject3.cfg : handled).
image

### CODE REVIEW

File: src/Core/Parser/Scene.cpp [hunk 1]
```diff
@@ -7,6 +7,42 @@
 
 #include "Parser/Scene.hpp"
 
+static bool filepathParsed(std::vector<std::string> parsedFiles, std::string filePath)
```
Comment (Njord201) [inline src/Core/Parser/Scene.cpp:10]: Rename the function to filePathParsed

File: tests/subject.cfg
Comment (Njord201) [inline tests/subject.cfg:1]: Remove the .cfg files

File: tests/subject2.cfg
Comment (Njord201) [inline tests/subject2.cfg:1]: Remove the .cfg files

File: tests/subject3.cfg
Comment (Njord201) [inline tests/subject3.cfg:1]: Remove the .cfg files

**Review by Njord201 — CHANGES_REQUESTED** [review]

File: tests/subject3.cfg
Comment (theophile-jr) [reply tests/subject3.cfg:1]: No.

File: tests/subject.cfg
Comment (theophile-jr) [reply tests/subject.cfg:1]: No.

Comment (Njord201) [inline tests/subject.cfg:1]: Rename the file for explain the test

File: tests/subject2.cfg
Comment (Njord201) [inline tests/subject2.cfg:1]: Rename the file for explain the test

File: tests/subject3.cfg
Comment (Njord201) [inline tests/subject3.cfg:1]: Rename the file for explain the test

**Review by Njord201 — CHANGES_REQUESTED** [review]

**Review by Njord201 — APPROVED** [review]
Good job ! :100:

**Review by AxelF44 — APPROVED** [review]

**Pull request merged**